src/qtiItem/core/Container.js

```diff
@@ -127,7 +127,10 @@ var Container = Element.extend({
                     //@todo : container rendering merging, to be tested
                     tpl = tpl.replace(elt.placeholder(), elt.render(renderer));
                 } else {
-                    tpl = tpl.replace(elt.placeholder(), '{{{' + serial + '}}}');
+                    tpl = tpl
+                        .replace(elt.placeholder(), serial)
+                        .replace('{', '&lcub;')
+                        .replace(serial, '{{{' + serial + '}}}');
```

Comment:
**suggestion:** The temporary replacement with the final value for transforming the remaining keywords is not robust. I'd rather suggest an approach where possible collision is avoided. Moreover, the `tpl` variable is shared for the whole loop and it may introduce issues when multiple elements are used. A better solution should be implemented.

Regarding the placeholder replacer, one could do this:
```suggestion
                    // Use a robust placeholder for replacing the element while converting entities
                    const tmp = `<!##${Math.floor(Math.random() * Math.pow(2, 32)).toString(16)}##!>`;
                    tpl = tpl
                        .replace(elt.placeholder(), tmp)
                        // Replace each occurrence of braces with computed HTML entities
                        .replace(/([{}])/gm, (match, group) => `&#${group.charCodeAt(0)};`)
                        .replace(tmp, '{{{' + serial + '}}}');
```

However, it will still create issues when having multiple elements in the same template, See: 
<img width="1055" alt="image" src="https://github.com/oat-sa/tao-item-runner-qti-fe/assets/1500098/2382fba0-b877-4e4c-b0af-5e3a4cf9d526">

**thought:** I'm not sure we can replace all braces this way. An approach would be to rewrite the loop, splitting it in several occurrences: a first loop to remove the elements with robust placeholders, another loop to replace the neutralize the braces, and finally a last loop to replace back elements with their serial... I'll let you figure out the implementation of this.

Reply:
Thanks for suggestions. I changed a bit approach of replacing.
As I see, serials is unique for each item and this replacement is happen for every item in container, so I not really sure about collision possibility.
Also important detail -- only open brace before placeholder breaks the flow